src/mono/wasm/runtime/loader/assets.ts

```diff
@@ -168,9 +164,12 @@ export async function mono_download_assets(): Promise<void> {
                             loaderHelpers.expected_instantiated_assets_count--;
                         }
                     } else {
-                        if (skipBufferByAssetTypes[asset.behavior]) {
-                            ++loaderHelpers.actual_downloaded_assets_count;
+                        if (asset.behavior === "symbols") {
+                            await runtimeHelpers.instantiate_symbols_asset(asset);
+                            cleanupAsset(asset);
                         }
+
+                        ++loaderHelpers.actual_downloaded_assets_count;
```

Comment:
I would prefer to keep the condition
`if (skipBufferByAssetTypes[asset.behavior])`

Reply:
the idea is that those *ByAssetTypes collections maintain what should happen, not the code/conditions. And in some cases that's not perfect. Feel free to improve it.